tests/copy-paste.js

```diff
@@ -630,6 +630,53 @@ require([
                    "double_trouble should not be valid");
         });
 
+        it("should not update broken references on paste into group", function () {
```

Comment:
Heh, good question. I think mainly because I only thought of it in the context of writing the next two tests. Also, currently copy-paste is not making a statement about how this is handled. There are many tests that could be written to confirm implementation details, but they're not necessary since there is no feature or bug that requires them.